src/asm/parser.y

```diff
@@ -1206,6 +1211,7 @@ reloc_16bit	: relocexpr {
 				warning(WARNING_TRUNCATION, "Expression must be 16-bit\n");
 			$$ = $1;
 		}
+		| inline_fragment { rpn_Symbol(&$$, $1); }
```

Comment:
Why not in `relocexpr`?

Reply:
That would allow inline fragments in a lot of places where they aren't really needed. If you want to do math on an address, just use a label for it.